PlaceRouteHierFlow/placer/ILP_solver.cpp

```diff
@@ -547,6 +552,12 @@ double ILP_solver::CalculateCost(design& mydesign, SeqPair& curr_sp) {
   cost += dead_area / area * const_graph.PHI;
   cost += linear_const * const_graph.PI;
   cost += multi_linear_const * const_graph.PII;
+  // constraint_penalty = cost - area_norm - HPWL_norm * const_graph.LAMBDA;
+  constraint_penalty =
```

Comment:
This is how we are computing the constraint_penalty now. Is seems to be zero all the time for cascoded_current_mirror_ota which I think means the ILP solver is working correctly.

I've excluded the dead_area / area term because that is not really a constraint issue, and I wonder why it is part of the cost function.